.vortex/installer/src/Utils/FileManager.php

```
@@ -31,11 +31,11 @@ public function prepareDestination(): array {
     }
 
     if (!is_readable($destination . '/.git')) {
-      $messages[] = sprintf('Initialising a new Git repository in directory "%s".', $destination);
+      $messages[] = sprintf('Initializing a new Git repository in directory "%s".', $destination);
       passthru(sprintf('git --work-tree="%s" --git-dir="%s/.git" init > /dev/null', $destination, $destination));
 
       if (!File::exists($destination . '/.git')) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'DrevOps\VortexInstaller\Utils\File' in method 'prepareDestination'. (undefined)

(StaticAccess)